core/src/utils/test/playwright/docs/usage-instructions.md

```diff
@@ -0,0 +1,106 @@
+# Usage Instructions
+
+E2E tests verify Ionic components in a real browser. This is useful for testing user interaction and catching visual regressions. We use Playwright as it allows us to test in multiple browsers. Tests can be written and run using Playwright's public API.
+
+## Table of Contents
+
+- [Installing Dependencies](#installing-dependencies)
+- [Running Tests](#running-tests)
+- [Managing Screenshots](#managing-screenshots)
+- [Further Reading](#further-reading)
+
+## Installing Dependencies
+
+Follow these steps to install Playwright dependencies. These steps must also be run whenever the installed version of Playwright changes so you test with the correct browser binaries.
+
+1. Install the Playwright dependency in the `core` directory: `npm ci` 
+2. Download the correct browsers: `npx playwright install`
+
+## Running Tests
+
+### Running All Test Files
+
+All E2E tests can be run using the following command:
+
+```shell
+npm run test.e2e
+```
+
+> [!NOTE]
+> This command is a wrapper for `npx playwright test`. All data passed to `npm run test.e2e` can also be passed to `npx playwright test`.
+
+### Running Specific Test Files
+
+Specific test files can be run by passing the file paths or a directory that contains multiple test files. See [Managing Screenshots](#managing-screenshots) for generating ground truths before running screenshot tests.
+
+**Specific Test Files**
+
+```shell
+npm run test.e2e src/components/button/test/basic/button.e2e.ts src/components/button/test/a11y/button.e2e.ts
+```
+
+**Test Directory with Multiple Files**
+
+```shell
+# Will run all the test files in the `test` directory
+npm run test.e2e src/components/button/test
+```
+
+## Managing Screenshots
+
+### Generating or Updating Ground Truths (Local Development)
+
+If you are running a test that takes a screenshot, you must first generate the screenshot as it renders in `main`. This is known as generating a "ground truth screenshot". All other screenshots will be compared to this ground truth. Alternatively, if `main` has changed since the last time you generated ground truths you may need to update your local ground truths.
```

Comment:
```suggestion
If you are running a test that takes a screenshot, you must first generate the reference screenshot from `main`. This is known as generating a "ground truth screenshot". All other screenshots will be compared to this ground truth. Alternatively, if `main` has changed since the last time you generated ground truths you may need to update your local ground truths.
```